core/logger.py

```diff
@@ -1,48 +1,43 @@
 #!/usr/bin/env python3
 
 from collections import Counter
-from datetime import datetime
 import os
 import sys
 
-import pandas as pd
 from absl import flags, app
+import pandas as pd
 
-from common import date_is_correct
+from common import date_is_correct, today
+import constants
 
 FLAGS = flags.FLAGS
-script_dir = os.path.dirname(os.path.abspath(__file__))
-repo_root_dir = os.path.join(script_dir, '..')
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+REPO_ROOT_DIR = os.path.join(SCRIPT_DIR, '..')
 
-flags.DEFINE_string('day', None, 'Day of the matchup in format yyyy-mm-dd. If not set, current day will be used')
-flags.DEFINE_string('winner', None, 'Name of the winner (player with lower rating value if drawn matchup is adding)')
-flags.DEFINE_string('looser', None, 'Name of the looser (player with greater rating value if drawn matchup is adding)')
-flags.DEFINE_string('score', None, "Result of match in W:L format (W - winner's won sets, L - looser's won sets)")
-flags.DEFINE_string('ball', 'yellow', 'Type of ball used for matchup')
-flags.mark_flags_as_required(['winner', 'looser', 'score'])
+flags.DEFINE_string('d', today(), 'Day of the matchup in format yyyy-mm-dd. If not set, today will be used.')
+flags.DEFINE_string('w', None, 'Name of the winner (player with lower rating value if drawn matchup is adding).')
+flags.DEFINE_string('l', None, 'Name of the looser (player with greater rating value if drawn matchup is adding).')
+flags.DEFINE_string('s', None, "Result of match in W:L format (W - winner's won sets, L - looser's won sets).")
+flags.DEFINE_string('b', 'yellow', 'Type of ball used for matchup. Available values: red, r, blue, b, yellow, y.')
+flags.mark_flags_as_required(['w', 'l', 's'])
 
 
 def check_player(player, games):
     known_players = set(games['Winner'].tolist() + games['Looser'].tolist())
     if player not in known_players:
         print('player {} has no mathups at database, Do you want to add him? ([y]/n)'.format(player))
         confirm = sys.stdin.readline().strip()
-        if confirm == 'n':
-            print('OK')
-            return False
-        elif confirm != 'y' and confirm != '':
-            print('Response is incorrect')
+        if confirm == 'n' or confirm not in ['y', '']:
```

Comment:
`confirm not in ['y', '']` and `confirm == 'n'` are equal, just keep one)

Reply:
What about incorrect responses?

Reply:
everything except `y` and `''` is `n`

Reply:
User can type everything, not only 'y', 'n' and '', for me such responses are incorrect (I abort them), see previous version of code.